ci/github/future-proofing.yaml

```diff
@@ -0,0 +1,83 @@
+name: Future Proofing
```

Comment:
@calinpristavu what do you think about the folder structure where we store these pipeline-related files (`./ci/github` and `./ci/gitlab`)?

Reply:
makes sense, not sure why we have to assume this structure for the consumers as well.

but people will decide on their own where they plant those ci files in their apps

Reply:
the structure for the consumers is there to avoid having weird file names for the two `future-proofing.yaml` files